### ISSUE TITLE
New icon

### ISSUE THREAD

**Issue by halcyonest** [SufficientlySecure/document-viewer#129]
![icondv](https://cloud.githubusercontent.com/assets/12594372/10511865/3b3b54fe-7377-11e5-9ba9-4ee6e82dcf0c.png)

*Download ![iconDV.svg.png](https://cloud.githubusercontent.com/assets/12594372/10511866/3b3f3ca4-7377-11e5-87cc-b53e8e5597d6.png) and remove '.png' extension to get an original SVG file.*

**Released under CC BY-NC**

**Comment by tuxor1337** [comment]
Looks nice. You are the author of this icon?

I can't decide on my own to change the icon of this app. It would be nice to have some feedback from other users. Furthermore, it would be nice to have some explanation from your side, why you think that document viewer needs a new icon and why the icon you provided is particularly suitable.

**Comment by dschuermann** [comment]
For reference: He also submitted to other projects like https://github.com/SufficientlySecure/offline-calendar/issues/47

I like this icon. Please check how it looks on other non-white backgrounds.

**Comment by halcyonest** [comment]
I think Document Viewer should prepare for Material Design. This icon is just a suggestion.

**Comment by frazzmark** [comment]
Could it be blue? I really like the old one, and this one looks very good, but I think that the new one should keep some aspects of the old one.

**Comment by hackerb9** [comment]
It looks nicer than the current icon, but I think the magnifying glass means "search" to a lot of people. What's the symbol for "reading"? Perhaps bifocal reading glasses or an open book?

**Comment by dschuermann** [comment]
bifocal reading glasses is a great idea, this is also what the KDE application Okular is using.

**Comment by halcyonest** [comment]
Here is a new icon. I didn't use bifocal reading glasses as a symbol of 'reader' beacuase its icon wasn't intuitive under Material Design language. The reason why I used green color as its main color is simply because I used it for other SufficientlySecure apps. 

![icondv2](https://cloud.githubusercontent.com/assets/12594372/11915931/f21dc628-a6ff-11e5-93e4-d66dee4548e1.png)

Download ![iconDV.svg.png](https://cloud.githubusercontent.com/assets/12594372/11915933/3f053962-a700-11e5-992a-84dcc7f524b9.png) and change its extension into '.svg' to get an original SVG file.

**Released under CC BY-NC-SA.**


Material Design is now a standard of Android apps. Is there any plans for implementing Material Design?

**Comment by hackerb9** [comment]
I like it. The decision to use a book instead of bifocals was a good one.

However, don't people use the "sheet of paper" icon to signify the documents an app can open (like an .epub or .pdf file) and then use a different icon for the app itself? Would it be possible to have the app icon be the book by itself but with horizontal lines on it and maybe a dog-eared page?

**Comment by ericwa** [comment]
Nice work! I like the "open book" part especially.

Agree with @hackerb9 that the book by itself could be nice. The green sheet of paper is quite close to the Google Docs icon, and I'm worried that drawing a connection with a word processor could increase the confusion that the project name already generates (people read "Document Viewer" and expect a .DOC/.DOCX viewer). For me anyway, the main use of the app is an ebook reader; DV is the best app for reading technical books in PDF format. I also use it for reading fiction in epub format.

Regarding Material Design, I did some first steps so the app uses the material theme on Android 5.0+. I also started work on replacing toolbar icons (haven't shared it in a pull request yet.)

**Comment by PanderMusubi** [comment]
Looking forward to Material Design icon for this app in 2017. Nice proposals. Please, continue implementing a new icon.

**Comment by rbrito** [comment]
Sorry if I am wrong here, but CC BY-NC-SA makes this icon non-free (the NC part). That being said, apart from the colors, I really like this icon better than the current (Tango?) icon.

**Comment by ericwa** [comment]
Yes, I guess we should avoid a -NC icon; F-Droid would flag it as an antifeature: https://f-droid.org/wiki/page/Antifeature:NonFreeAssets

I'm not sure if @ghost 's account is still around, and sorry for asking this 2 years late but would you be OK with re-licensing the icon as Apache 2.0?

**Comment by dschuermann** [comment]
ghost is just GitHub's way of saying that the account has been deleted :(

In another thread he agreed to re-license to Apache: https://github.com/PrivacyApps/offline-calendar/issues/47
